src/java.desktop/share/classes/javax/swing/Box.java

```diff
@@ -425,6 +430,11 @@ public AccessibleContext getAccessibleContext() {
      */
     @SuppressWarnings("serial")
     protected class AccessibleBox extends AccessibleAWTContainer {
+        /**
+         * Constructs a {@code AccessibleBox}.
+         */
+        public AccessibleBox() {}
```

Comment:
Shouldn't this be a protected constructor?
"if the class is declared protected, then the default constructor is implicitly given the access modifier protected".

Reply:
Modified to "protected" constructor

Reply:
These classes are not abstract so not sure about the spec for the constructor: "Constructor for subclasses to call." Some of them like "AccessibleBox" is instantiated directly.

Reply:
Let's see what Joe says about the CSR spec...I will change if needed.